supermarq-benchmarks/supermarq/benchmark.py

```diff
@@ -19,14 +21,25 @@ class Benchmark:
     and compute the final score.
     """
 
+    def cirq_circuit(self) -> cirq.Circuit | list[cirq.Circuit]:
+        """Returns the cirq circuit(s) corresponding to the current benchmark parameters."""
+        return self.circuit()
+
     @abc.abstractmethod
-    def circuit(self) -> cirq.Circuit | Sequence[cirq.Circuit]:
-        """Returns the quantum circuit corresponding to the current benchmark parameters."""
+    def circuit(self) -> cirq.Circuit | list[cirq.Circuit]:
```

Comment:
nit: can we move this either up or down so `cirq_circuit` and `qiskit_circuit` are adjacent?

Reply:
Yeah, that would be nicer -- done